### DISCUSSION TITLE
WorkNotes Management

### DISCUSSION THREAD

**Discussion by TreeWoman** [windingwind/zotero-better-notes#889] [Q&A]
1. It seems I can have a workspace note for each collection - which is great. But - if I mess up a Workspace Note while I am still learning - there seems to be no way to delete its content, or ultimately the note itself and start over. What am I missing here?

2. Also, according to one discussion,  - there is a "File > Open Recent Worknote"  available, but in my up-to-date versions of BN and Zotero it is under neither File menu. It would be great to have a menu item of all the worknotes currently available - to make it easy to switch back and forth.

Thanks in advance for your help and support!

**Comment by BorgeJorge** [comment]
Agreed, would love to know how to delete a note workspace.

**Comment by windingwind** [comment]
Select the note in library and delete it.
